src/routes/analytics/DashboardPage.tsx

```diff
@@ -15,7 +15,11 @@ export function DashboardPage(props: Props) {
 
   return (
     <Box mih="100vh" className="dashboard-container smartscalar">
-      <InteractiveDashboard dashboardId={dashboardId} withTitle withDownloads />
+      <InteractiveDashboard
+        dashboardId={dashboardId}
+        withTitle
+        withDownloads={false}
```

Comment:
This is also hiding results (json,csv etc) downloads right?
After we look into the issue tomorrow we should talk with Alberto to see if we should split the props in two.
I also expect pngs to be broken so let's also check that

Reply:
@npretto yes, that also hides the "download result" button in each card. I think splitting in two makes sense.